robin-irc.php

```diff
@@ -386,15 +387,25 @@ public function __call($name, $args) {
 
                     case 'JOIN':
                         $channels = explode(',', $args[0]);
+                        $names = array();
+                        foreach ($this->users as $k => $user) {
```

Comment:
You could probably do this with an array_map, but it's not like it matters any more...